main/src/com/google/refine/grel/ast/FieldAccessorExpr.java

```diff
@@ -62,14 +63,17 @@ public Object evaluate(Properties bindings) {
         if (ExpressionUtils.isError(o)) {
             return o; // bubble the error up
         } else if (o == null) {
-            return null;
+            if ("value".equals(_fieldName)) {
+                return null;
+            }
+            return new EvalError("Cannot retrieve field from null");
         } else if (o instanceof HasFields) {
             return ((HasFields) o).getField(_fieldName, bindings);
         } else if (o instanceof ObjectNode) {
-        	JsonNode value = ((ObjectNode) o).get(_fieldName);
-        	return JsonValueConverter.convert(value);
+            JsonNode value = ((ObjectNode) o).get(_fieldName);
+            return JsonValueConverter.convert(value);
         } else {
-            return null;
+            return new EvalError("No accessible fields available");
```

Comment:
Hmm, I wonder if we can make this wording more noob friendly? Something a bit more meaningful to a beginning data journalist or librarian learning GREL.  What does "accessible even mean", what is a "field"?  Our product reference will have to explain this deeply.

Reply:
Feel free to offer a suggestion based on your experience with users who are just starting out (which I assume is what you meant rather than "noob").

Reply:
Sorry, was in the middle of a PC game while I was looking at this ... "lingo" within that domain. :-) 
I'm not sure what I would say either.  Hmm... let me think on it.  "field" has been the main thing that made heads spin, and wondering if we could change that term somehow.

Reply:
Hmm, variable? a modifier? an attribute? https://en.wikipedia.org/wiki/Field_(computer_science)  https://en.wikipedia.org/wiki/Instance_variable

Reply:
```suggestion
            return new EvalError("You have asked for a non-existent field from the object or tuple, or there is no output for your expression. You might want to double check things, or ensure your working with a valid cell/row object or tuple");
```

I think since this is Evaluable, we should say something about the output of the expression.  I mentioned something like the above suggestion in #1036 and have made more concise.